mappings/net/minecraft/network/packet/c2s/play/ChatMessageC2SPacket.mapping

```diff
@@ -14,8 +14,18 @@ CLASS net/minecraft/class_2797 net/minecraft/network/packet/c2s/play/ChatMessage
 	COMMENT @see net.minecraft.server.network.ServerPlayNetworkHandler#onChatMessage
 	FIELD field_12764 chatMessage Ljava/lang/String;
 	FIELD field_33359 MAX_LENGTH I
+	FIELD field_39086 TWO_MINUTES Ljava/time/Duration;
+	FIELD field_39087 instant Ljava/time/Instant;
 	METHOD <init> (Ljava/time/Instant;Ljava/lang/String;Lnet/minecraft/class_3515$class_7425;)V
+		ARG 1 time
```

Comment:
> This parameter should be consistent with `field_39087` and `method_43634`.

My previous comment was marked as resolved, but the issue is not addressed. These three should get the same name.

Reply:
Whoops, sorry )